operator/BACKPORTING.md

```diff
@@ -0,0 +1,76 @@
+# Backporting upstream PRs into an OpenShift Logging Release Branch
+
+The following instructions describe how to backport a merged Loki-Operator upstream PR into an OpenShift Logging release branch.
+
+## Background
+
+OpenShift Logging releases branches for the Loki Operator follow the product version scheme. In detail each minor version of the OpenShift Logging 5 release streams yield individual `release-5.y` release branches. For example, OpenShift Logging `5.5.0 - 5.5.z` expects that the bundled operators are build from a corresponding branch named `release-5.5`.
+
+## Instructions
+
+The following approach describes how to backport an upstream PR merged into the [grafana/loki][grafana-loki] `main` branch.
+
+__Note:__ The grafana/loki upstream repository follows a squash-and-merge approach when merging PRs. This means each PR represents one commit in the `main` branch history. In case you had to create follow-up PRs for the same issue because of flaws or incompleteness identified after merge, you are encouraged to combine them in a **single** backport PR.
```

Comment:
```suggestion
__Note:__ The grafana/loki upstream repository follows a squash-and-merge approach when merging PRs. This means each PR represents one commit in the `main` branch history. In case you had to create follow-up PRs for the same issue because of flaws or incompleteness identified after the merge, you are encouraged to combine them in a **single** backport PR.
```